Add line breaks between multiple error/warning messages in debug console

fisher-alice opened the pull request:
## Summary

- There are line breaks between multiple error/warning messages when hovering the gutter icon, but prior to this update, line breaks were not included in these messages in the Debug Console. 

<img width="878" alt="image-20220726-223248" src="https://user-images.githubusercontent.com/107423305/184964662-d8a09732-ef9f-4606-bb7a-6a30baecc5cb.png">
<img width="663" alt="onEvent before line break" src="https://user-images.githubusercontent.com/107423305/184965034-462a7236-4bfa-4ec0-a4f1-8baadecdfcb3.png">

- Added line breaks to increase readability when there are multiple errors/warnings displayed in Debug Console.
<img width="502" alt="onEvent after line break" src="https://user-images.githubusercontent.com/107423305/184965205-73983151-4e7d-4c2b-98bf-dbcd80caf3d5.png">

- Based on Mike's feedback, ensured update didn't add an extra line break for gutter-line warnings/errors.

<img width="1250" alt="revised code" src="https://user-images.githubusercontent.com/107423305/184995249-cbbb3643-9ae3-44fe-8691-fd3f524a74cf.png">


## Links

- [jira ticket](https://codedotorg.atlassian.net/browse/STAR-2355)

mikeharv reviewed (changes requested):
Thanks for tackling this! I believe this unfortunately introduces a regression that we want to avoid. I pulled and tried out your changes. What I found is that the line break work as expected in the user's console but they are also added to the warnings and errors when hovering over the corresponding symbols in the gutter: 

![image](https://user-images.githubusercontent.com/43474485/184967775-63cbd2cd-468e-4898-bf54-80e66a58cc21.png)

I think the right solution here will involve digging down a layer deeper so that we only add the new line when printing to the console. In other words, `errorString` seems to have the right format, but how we're using varies between the two cases.

breville commented:
@mikeharv Excellent catch!

fisher-alice commented:
> Thanks for tackling this! I believe this unfortunately introduces a regression that we want to avoid. I pulled and tried out your changes. What I found is that the line break work as expected in the user's console but they are also added to the warnings and errors when hovering over the corresponding symbols in the gutter:
> 
> ![image](https://user-images.githubusercontent.com/43474485/184967775-63cbd2cd-468e-4898-bf54-80e66a58cc21.png)
> 
> I think the right solution here will involve digging down a layer deeper so that we only add the new line when printing to the console. In other words, `errorString` seems to have the right format, but how we're using varies between the two cases.

Thanks so much for catching this! - updated code so extra line breaks are not added to gutter-line mesages
<img width="1250" alt="revised code" src="https://user-images.githubusercontent.com/107423305/184995777-03865164-f3ff-4fad-ac77-78d836247281.png">
.

mikeharv reviewed (approved):
Nice!
